test/multiowned.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
+
+// testrpc has to be run as testrpc -u 0 -u 1 -u 2 -u 3
+
+const expectThrow = require('./helpers/expectThrow');
+
+const multiowned = artifacts.require("./multiowned.sol");
```

Comment:
Can we use `Holder` instance instead? They should be compatible, but we will be sure we are testing our business object, not the underlying implementation which we can override or replace.